extensions/amp-live-list/0.1/amp-live-list.js

```diff
@@ -664,6 +698,10 @@ export class AmpLiveList extends AMP.BaseElement {
     for (let child = updatedElement.firstElementChild; child;
       child = child.nextElementSibling) {
       const id = child.getAttribute('id');
+      const sortTime = child.getAttribute('data-sort-time');
```

Comment:
can we add an explainer here how an item can have no sort time (since data-sort-time is mandatory in spec)

Reply:
Done.